src/tests.rs

```diff
@@ -99,3 +99,36 @@ fn last() {
     assert_last_eq!(set, None);
     assert_eq!(set.pop_last(), None);
 }
+
+#[test]
+fn test_open_ranges() {
```

Comment:
```suggestion
fn from_open_ranges() {
```

This name is a bit more clear as to what's being tested.